x-pack/plugins/lists/common/schemas/types/comment.mock.ts

```diff
@@ -4,14 +4,15 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { DATE_NOW, USER } from '../../constants.mock';
+import { DATE_NOW, ID, USER } from '../../constants.mock';
 
-import { Comments, CommentsArray } from './comments';
```

Comment:
Made these singular, was really confusing when it was plural but referred to a single comment schema.